ansible/configs/ocp4-workshop/post_software.yml

```diff
@@ -395,5 +395,27 @@
   gather_facts: false
   become: false
   tasks:
+  - name: Get kubeadmin password
+    slurp:
+      path: /home/{{ ansible_user }}/{{ cluster_name }}/auth/kubeadmin-password
+    register: kubeadminr
+    when: report_status | d(false)
```

Comment:
We could make block so that these tasks all fall under the same `when: report_status` condition